sw/device/tests/rv_core_ibex_epmp_test.S

```diff
@@ -0,0 +1,56 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/silicon_creator/manuf/lib/sram_start.h"
+
+  /**
+   * The User Mode test function.
+   *
+   * This will be the only area, other than test location(s),
+   * that will be executable by the user.
```

Comment:
Can you add a comment explaining why this was written in assembly?
As this PR has already passed CI, it's ok to do it in a follow-up PR.